pkg/util/util_test.go

```diff
@@ -281,3 +289,49 @@ func TestComponentsPausedForMigration(t *testing.T) {
 	cluster.Annotations[constants.AnnotationPauseComponentMigration] = "true"
 	require.True(t, ComponentsPausedForMigration(cluster))
 }
+
+func TestGetTopologySpreadConstraints(t *testing.T) {
```

Comment:
Can we add a quick test for HaveTopologySpreadConstraintsChanged as well just to keep the unit test coverage up? I know this is tested from the individual components, but go test doesn't count it in the coverage.